index.html

```diff
@@ -8,7 +8,7 @@
 	<h1>Background Generator</h1>
 	<input class="color1" type="color" name="color1" value="#00ff00">
 	<input class="color2" type="color" name="color2" value="#ff0000">
-	<h2>Display CSS Background Code</h2>
+	<h2>This is the backgrount</h2>
```

Comment:
Hey Ola, this looks good, it has a spelling error ("backgrount") instead of "background".
I'll work on that.
Thanks.